modules/howtos/examples/user-manager.js

```diff
@@ -39,6 +39,10 @@ async function example1(clusterAdm, testUsername, testPassword, bucketName) {
         ]
     })
     // end::usermanagement_1[]
+
+    // Give some time (2 seconds) for user to be created as example code
+    // might run too quickly.
+    await new Promise((resolve) => setTimeout(resolve, 2000))
```

Comment:
Is there a JSCBC report for these 2 new slowups?
Race condition in testing doesn't feel great.

Reply:
No, maybe worth asking at the Friday meeting this, as they probably would have performance tested the SDK? 
Could be more a "hey we noticed this took a fraction longer, maybe you've seen this as well?"

Reply:
Yeah, I guess *quantitative* slowups isn't any of our business...

BUT: immediate -> takes time, is a qualitative change isn't it?
It feels a bit like a new behaviour change (as we have to add poll/race-sleep to workaround)